webpack/assets/javascripts/react_app/redux/API/__tests__/__snapshots__/APISelectors.test.js.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`API selectors should return the API substate by key 1`] = `
+Object {
+  "controller": "some_controller",
+  "error": null,
+  "status": "RESOLVED",
+}
+`;
+
+exports[`API selectors should return the API substate controller 1`] = `"some_controller"`;
```

Comment:
Didn't you remove the "substate"?

Reply:
in `substate` I mean the scope of the key:
```js
{
...store,
API:{
  key: { // this is the substate
     results, status, error
}
}
}
```